src/backend/db/products.js

```diff
@@ -3,8 +3,8 @@ export const products = [
   {
     _id: uuid(),
     productName: "Lemon",
-    price: "₹10",
-    prevPrice: "₹20",
+    price: "₹5",
+    prevPrice: "₹10",
     imgSrc: "assets/products/lemon.png",
```

Comment:
would suggest to host these images on a CDN instead of keeping it here.
Would increase the bundle size as the amount of products grow

Reply:
Changes done. Thank you for this suggestion